### DISCUSSION TITLE
how reset input type color appearance with tailwind?

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#14428] [Help] [answered]
how can we remove this gray border around 
`<input type="color">`
in tailwind?
![image](https://github.com/user-attachments/assets/0067b844-a6cb-48b8-8954-87f97cb6874b)

**Comment by YuHyeonWook** [comment, marked as answer]
The default styles for HTML elements, like the gray border around <input type="color">, are provided by the browser. Therefore, these styles cannot be completely removed with Tailwind CSS alone.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

**Reply by MonaAghili** [reply]
Thank you
